<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_titlecheck</class>
 <widget class="QWidget" name="w_titlecheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>1011</width>
     <height>681</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <column>
    <property name="text">
     <string>Title</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Character</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Error</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Image</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnLoad">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>710</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>710</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReduceSize">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>710</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Reduce size</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOrginalSize">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>710</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Original Size</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>760</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbWait">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>740</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Please wait</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>720</y>
     <width>281</width>
     <height>16</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Total :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbTotal">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label2">
      <property name="text">
       <string>Error : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbError">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Percentage :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbPercentage">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>904</x>
     <y>710</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Export to excel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
